plugins/cube/skydome.cpp

```diff
@@ -139,11 +139,14 @@ void wf_cube_background_skydome::render_frame(const wf_framebuffer& fb,
     fill_vertices();
     reload_texture();
 
-    if (tex == (uint32_t)-1)
+    if (tex == (uint)-1)
```

Comment:
I generally prefer to use the full `uint32_t`

Reply:
I went ahead and changed all instances of `uint` to `uint32_t` in the cube plugin.